File: Poin B.php/Memory.php
<?php

//Include file Parent
include "Hardware.php";

class Memory extends Hardware{
    //Deklarasi Atribut
    private $frequency;
    private $memorySize;
    private $supportCuda;



    //Constructor
	function __construct() {
	}
	
    //Get Set Methods
	function getFrequency() {
		return $this->frequency;
	}

	function setFrequency($frequency){
		$this->frequency = $frequency;	
	}

	function getMemorySize() {
		return $this->memorySize;
	}
	
	function setMemorySize($memorySize){
		$this->memorySize = $memorySize;
	}

	function getSupportCuda() {
		return $this->supportCuda;
	}

	function setSupportCuda($supportCuda){
		$this->supportCuda = $supportCuda;
	}

    //Output Methods
    function output_memory(){
        $this->output_product();
        $this->output_hardware();
        echo "Frequency				: ". $this->getFrequency()."<br/>";
		echo "Memory Size			: ". $this->getMemorySize()."<br/>";
		echo "Support Cuda			: ". $this->getSupportCuda()."<br/>";

    }


    //Destructor
    function __destruct()
    {
        
    }
}
?>

File: Poin B.py/Memory.py
#import kelas parent
from Hardware import Hardware


class Memory(Hardware) :
    #inisialisasi atribut
    __Frequency = ""
    __MemorySize = ""
    __SupportCuda = ""
    
    
    #constructor
    def __init__(self, freq = "", size = "", Cuda = "") :
        self.__Frequency = freq
        self.__MemorySize = size
        self.__SupportCuda = Cuda
        
    #getset methods
    def setFrequency(self, freq = "") :
        self.__Frequency = freq
        
    def getFrequency(self) :
        return self.__Frequency
    
    def setMemorySize(self, size = "") :
        self.__MemorySize = size
        
    def getMemorySize(self) :
        return self.__MemorySize
    
    def setSupportCuda(self, cuda = "") :
            self.__SupportCuda = cuda
        
    def getSupportCuda(self) :
        return self.__SupportCuda
    
    #output method
    def outputMemory(self):
        self.outputProduct()
        self.outputHardware()
        print("Frequency                     : "+ str(self.getFrequency()))
        print("Memory Size                   : "+ str(self.getMemorySize()))
        print("Support Cuda                  : "+ str(self.getSupportCuda()))
        
    
        
    